components/toast/toast.html

```diff
@@ -0,0 +1,120 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+    />
+
+    <link rel="stylesheet" href="../../style.css" />
+    <title>Toast</title>
+  </head>
+  <body>
+    <div class="container">
+      <nav>
+        <div class="left-nav">
+          <a href="/index.html">Aqua UI</a>
+        </div>
+        <ul class="right-nav">
+          <li><a href="#">Documentation</a></li>
+          <li><a href="#">Examples</a></li>
+        </ul>
+      </nav>
+
+      <main>
+        <h1 class="main-heading">Toast</h1>
+        <h4 class="heading-info">
+          Toasts are usually used to put out a notification for the user. It can be even called as a notify bar and snack bar.
+        </h4>
+        <h3 class="sub-heading">
+          Below is an example for toast.
+        </h3>
+        <section class="component">
+          <div class="toast">
+              <button class='btn-toast'>A toast message  <i class="fas fa-times icon-dismiss"></i></button>
```

Comment:
your whole toast is wrapped in the btn, which should be in div. 
for reference you can check https://getbootstrap.com/docs/4.3/components/toasts/

Reply:
sure will make the changes. thank you